packages/starrating/src/css/index.js

```diff
@@ -0,0 +1,17 @@
+export default {
+  button: {
```

Comment:
We use BEM in our css files because we export stylesheets.  For an example, please see:

https://github.com/pluralsight/design-system/blob/master/packages/button/src/css/index.js#L16

Reply:
Would you happen to know of a class component that does this? I'm a bit confused by what's going on.